number-systems/README.md

```diff
@@ -5,61 +5,61 @@ Do not convert any binary numbers to decimal when solving a question unless the
 The goal of these exercises is for you to gain an intuition for binary numbers. Using tools to solve the problems defeats the point.
 
 Convert the decimal number 14 to binary.
-Answer:
+Answer: 1110.
```

Comment:
Could you please submit this as a separate PR if possible? One task per pr, I think jq was also a separate one :)